package/gluon-web-model/luasrc/usr/lib/lua/gluon/web/model/classes.lua

```diff
@@ -361,6 +361,66 @@ function ListValue:validate()
 end
 
 
+local MultiListValue = class(AbstractValue)
+M.MultiListValue = MultiListValue
+
+function MultiListValue:__init__(...)
+	AbstractValue.__init__(self, ...)
+	self.subtemplate  = "model/mlvalue"
+
+	self.size = 1
+
+	self.keys = {}
+	self.entry_list = {}
+end
+
+function MultiListValue:value(key, val, ...)
+	key = tostring(key)
+
+	if self.keys[key] then
+		return
+	end
+	self.keys[key] = true
+
+	val = val or key
+	table.insert(self.entry_list, {
+		key = key,
+		value = tostring(val),
+		deps = {...},
+	})
+end
+
+function MultiListValue:entries()
+	local ret = {unpack(self.entry_list)}
+
+	return ret
+end
+
+function MultiListValue:validate()
+	for _, val in ipairs(self.data) do
+		if not self.keys[val] then
+			return false
+		end
+	end
+
+	return true
+end
+
+function MultiListValue:defaultvalue()
+	local value = self.default
+
+	if type(value) == "table" then
+		return value
+	else
+		return { value }
+	end
```

Comment:
It would be better to disallow non-table default values instead of fixing them up (same for `DynamicList`, where this was copied from)

Reply:
https://github.com/freifunk-gluon/gluon/blob/670da836c7c75769dd7abbf87fde66b713cdf2a1/package/gluon-web-model/luasrc/usr/lib/lua/gluon/web/model/classes.lua#L372-L380

Reply:
We can still allow `nil` for convenience, making this something like `return self.default or {}`.